data/json/furniture_and_terrain/furniture-terrains.json

```
@@ -426,63 +440,69 @@
     "description": "This canvas flap door has been pulled aside.",
     "symbol": ".",
     "color": "blue",
-    "move_cost_mod": 0,
+    "move_cost_mod": 2,
+    "coverage": 95,
     "required_str": -1,
-    "flags": [ "TRANSPARENT" ],
+    "flags": [ "FLAMMABLE", "INDOORS", "TRANSPARENT" ],
     "close": "f_canvas_door",
     "bash": {
       "str_min": 1,
       "str_max": 8,
       "sound": "rrrrip!",
       "sound_fail": "slap!",
       "sound_vol": 8,
-      "tent_centers": [ "f_groundsheet", "f_fema_groundsheet", "f_skin_groundsheet", "f_center_groundsheet" ]
-    }
+      "//": "Go straight through without taking any damage.",
+      "ranged": { "block_unaimed_chance": "100%" }
+    },
+    "deconstruct": { "items": [ { "item": "sheet", "count": 2 } ] }
   },
   {
     "type": "furniture",
-    "id": "f_large_canvas_door",
-    "name": "canvas flap",
-    "description": "This heavy canvas flap door could be pulled aside.",
+    "id": "f_large_canvas_door_c",
```

```suggestion
    "id": "f_large_canvas_door",
```
You have references to f_large_canvas_door elsewhere, so either need this change or replacing all the other references then obsoleting the f_large_canvas_door into f_large_canvas_door_c would be required for save file compatibility